### ISSUE TITLE
Language selection in CMS and frontend simultaneously (Serious Problem)

### ISSUE THREAD

**Issue by ghost** [tractorcow-farm/silverstripe-fluent#20]
Hi Damian!

If I open two tabs in browser.
In the first tab open the CMS and select a record of a class that inherits from DataObject (ModelAdmin) and select the English language.
In the second tab I open the website that I'm creating and select another language with its language selector, for example, Spanish.
Now I select CMS tab and I click on the 'Save' button in the CMS.
In that record that I saved, the English language texts overwrites the Spanish language texts of the multilanguage fields.

In summary, the cookie for the locale that you use in the CMS should be different to that used in the frontend, since side effects may occur.
You will need to include a current_locale() method for CMS and a different one for the FrontEnd.

Regards,
Jose A.
